server/test/unit/server/agent/direct/test_agent.py

```diff
@@ -61,243 +78,201 @@ def test_delete_queue_not_found(self, queue):
         agent.delete_queue(url, name)
 
         # validation
-        queue.assert_called_once_with(name)
+        add_connector.assert_called_once_with()
+        queue.assert_called_once_with(name, url)
         queue = queue.return_value
-        queue.purge.assert_called_once_with(url)
+        queue.purge.assert_called_once_with()
         self.assertFalse(queue.delete.called)
 
     @patch('pulp.server.agent.direct.pulpagent.Agent')
-    def test_unregistered(self, mock_gofer_agent):
-        context = Mock()
-        context.route = '123'
-        context.secret = 'test-secret'
-        context.url = 'http://broker.com'
-        context.authenticator = Mock()
-
-        mock_agent = Mock()
-        mock_gofer_agent.return_value = mock_agent
-        mock_consumer = Mock()
-        mock_agent.Consumer = Mock(return_value=mock_consumer)
-
-        # test capability
+    def test_unregistered(self, _agent):
```

Comment:
I think this should be renamed similar to the function itself being renamed to `unregister()`